### PR TITLE
add vcpkg to readme

### LINKED ISSUE #853: Port for vcpkg
Hello,
I've opened an issue in the [VCPKG repo[]( https://github.com/microsoft/vcpkg/issues/38066) to get this nice library integrated in the packaging system that i use most often. It would be very convenient to have Manifold supported by vcpkg. 
If any one feel like he can do it that would be super nice.
If noone here can do it soon, I will try to do it myself but that would be in a month or so.
PS : i'm also using manifold through the python binding, i may later open some issues we have met so far.
Regards !

### PR DESCRIPTION
**Pull request by JeffreyWardman**
Fixes #853

### CODE REVIEW

**Comment by pca006132** [comment]
I think maybe we should add it to the bindings and packages section?

**Comment by JeffreyWardman** [comment]
Updated. Let me know if the new commmit suits.

**Comment by pca006132** [comment]
LGTM, what do you think @elalish ?

File: README.md [hunk 1]
```diff
@@ -24,11 +24,12 @@ Here is an incomplete list of our users, whose integrations may be anywhere from
 
 ### Bindings & Packages
 
-If C++ isn't your cup of tea, Manifold also has bindings to many other languages, some maintained in this repository, and others elsewhere.
+Manifold has bindings to many other languages, some maintained in this repository, and others elsewhere. It can also be built in C++ via [vcpkg](https://github.com/microsoft/vcpkg.git).
 
 | Language | Packager | Name | Maintenance |
 | --- | --- | --- | --- |
 | C | N/A | N/A | internal |
+| C++ | vcpkg | [manifold](https://github.com/microsoft/vcpkg/tree/master/ports/manifold) | internal |
```
Comment (elalish) [inline README.md:32]: This says vcpkg is under internal maintenance - are we maintaining this package or are you? If it's really going to be internal, we could use a hand setting up a Github Action to publish it.

**Review by elalish — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pca006132) [reply README.md:32]: I don't think it can be internal, we need to get vcpkg repo to approve the PR.

Comment (pca006132) [reply README.md:32]: Anyway, nice catch :)

Comment (JeffreyWardman) [reply README.md]: Updated to external. Misunderstood it to mean bindings written by others. Thanks!

**Pull request merged**